ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -4235,6 +4235,40 @@ def test_torch_tensor_bernoulli(
     )
 
 
+# bernoulli_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bernoulli_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+    test_with_out=st.just(True),
```

Comment:
Remove this flag

Reply:
Let me know if that works

Reply:
@umairjavaid you mean removing "test_with_out=st.just(True)" line right?